nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/authorizers.xml

```diff
@@ -198,12 +198,22 @@
             NOTE: Any identity mapping rules specified in nifi.properties are not applied in this implementation. This behavior
             would need to be applied by the base implementation.
     -->
+
+    <!-- To enable the shell-user-group-provider remove 2 lines. This is 1 of 2.
+    <userGroupProvider>
+        <identifier>shell-user-group-provider</identifier>
+        <class>org.apache.nifi.authorization.ShellUserGroupProvider</class>
+        <property name="Initial Refresh Delay">30</property
```

Comment:
I think we would prefer these values to include the units like the LDAP timeouts above: https://github.com/apache/nifi/pull/3495/files#diff-8f577f792e128744ad8f8f5eea585ac4R144

Reply:
Here is the LDAP code for parsing those values into the proper numerical value given the time units: https://github.com/apache/nifi/blob/531c84ac31f93ef72521134a7cc5ea19a2f1e276/nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers/src/main/java/org/apache/nifi/ldap/tenants/LdapUserGroupProvider.java#L368

Reply:
Great idea.  Added support for unit formatting.